README.md

```diff
@@ -44,6 +44,11 @@ cargo install diesel_cli --no-default-features --features postgres
 
 After you have installed the CLI, you can run the following from the root of the project:
 
+HINT: The format for <DATABASE_URL> is:
+```bash
+postgres://username:password@localhost:5432/database_name
```

Comment:
Can we move this under the command and have verbiage such as "where `<DATABASE_URL>` uses the `pogtgres://` URI. Example: `postgres://username:password@localhost:5432/database_name`